milli/src/update/index_documents/typed_chunk.rs

```diff
@@ -177,6 +179,26 @@ pub(crate) fn write_typed_chunk_into_index(
             )?;
             is_merged_database = true;
         }
+        TypedChunk::GeoPoints(mut geo_points) => {
+            // TODO: we should create the rtree with the `RTree::bulk_load` function
```

Comment:
Do we want to keep this TODO here? In which case don't we want to create an issue about that? Do we wait for guigui to try the feature first?

Reply:
I would say we keep the TODO, and we create an issue once it's merged :thinking: 
But as you want, I don't know the rules about letting `TODO`s hanging in the code :grin:

Reply:
That's not really something that we do a lot, I would prefer that we just open an issue, please :)